pom.xml

```diff
@@ -11,6 +11,7 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
To fix a warning when building the app

Reply:
Nice 💯  I was not sure how to fix theses warnings